media.module

```diff
@@ -22,3 +24,46 @@ function media_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function media_form_alter(&$form, FormStateInterface &$form_state) {
+  if ($form['#form_id'] == 'entity_browser_media_library_form') {
+    // Attach our library.
+    $form['#attached']['library'][] = 'media/view';
+
+    // Style the submit button.
+    $form['actions']['submit']['#value'] = t('Select Media Files');
+    $form['actions']['submit']['#attributes']['class'][] = 'button--primary';
+    $form['actions']['submit']['#attributes']['class'][] = 'entity-browser-modal-target';
+
+    // Add a class for generic styling.
+    $form['#attributes']['class'][] = 'media-library-form';
+  }
+}
+
+/**
+ * Implements hook_library_info_alter().
+ */
+function media_library_info_alter(&$libraries, $extension) {
+  // Optionally use the Libraries module to determine our library paths.
+  if ($extension == 'media' && \Drupal::moduleHandler()->moduleExists('libraries')) {
+    $imagesloaded_path = libraries_get_path('imagesloaded') . '/imagesloaded.pkgd.min.js';
+    $masonry_path = libraries_get_path('masonry') . '/dist/masonry.pkgd.min.js';
+
+    $libraries['imagesloaded']['js'] = ['/' . $imagesloaded_path => ['minified' => 'true']];
+    $libraries['masonry']['js'] = ['/' . $masonry_path => ['minified' => 'true']];
+  }
+}
+
+/**
+ * Implements hook_views_pre_render().
+ *
+ * Adds the media.user library to the media_library view.
+ */
+function media_views_pre_render(ViewExecutable $view) {
+  if (isset($view) && ($view->storage->id() == 'media_library')) {
+    $view->element['#attached']['library'][] = 'media/user';
```

Comment:
Okay I am doing that. File entity browser had different css files, that's why I made it like that.